### PR TITLE
Feature/UI elements

### PR DESCRIPTION
**Pull request by InvisibleHorse**
Created a list of cards

### CODE REVIEW

File: src/components/monsters/Image/ImageOfMonster.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CardMedia from '@mui/material/CardMedia';
```
Comment (w00tc) [inline src/components/monsters/Image/ImageOfMonster.jsx:2]: react prop types redux and functional stuff always at the top of the import

File: src/components/monsters/Monster.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
```
Comment (w00tc) [inline src/components/monsters/Monster.jsx:1]: ?

File: src/components/monsters/Monster.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { Button, CardActions } from '@mui/material';
+import PropTypes from 'prop-types';
```
Comment (w00tc) [inline src/components/monsters/Monster.jsx:6]: import order

File: src/components/monsters/Monster.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import ImageOfMonster from './Image/ImageOfMonster';
+import s from './Monsters.module.css';
```
Comment (w00tc) [inline src/components/monsters/Monster.jsx:8]: enter

File: src/components/monsters/Monster.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+export default function Monster(props) {
+    return (
+        <div className={s.zoom}>
+            <Card className={s.card} sx={{ width: 250 }}>
```
Comment (w00tc) [inline src/components/monsters/Monster.jsx:13]: you have a class, why do you need inline styles?

File: src/components/monsters/Monster.jsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+                <ImageOfMonster id={props.id} />
+                <CardContent className={s.cardContent}>
+                    <Typography gutterBottom variant="h5" component="div">
```
Comment (w00tc) [inline src/components/monsters/Monster.jsx:16]: I don't see theme.js, you should learn how to globally work with styles in MUI. How variant works

File: src/components/monsters/Monsters.jsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-named-as-default,import/no-named-as-default-member
```
Comment (w00tc) [inline src/components/monsters/Monsters.jsx:3]: remove

File: src/containers/monsters/SingleMonsterContainer.jsx [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Monster from '../../components/monsters/Monster';
+
+export default function SingleMonsterContainer(props) {
```
Comment (w00tc) [inline src/containers/monsters/SingleMonsterContainer.jsx:4]: it's a container that don't have any sideEffects and dependencies, I don't see any useSelector, useDispatch, async code. So it's just a component

**Review by w00tc — CHANGES_REQUESTED** [review]

File: src/components/monsters/Monster.jsx (hunk 4 above)
Comment (InvisibleHorse) [reply src/components/monsters/Monster.jsx:8]: ?

Comment (InvisibleHorse) [reply src/components/monsters/Monster.jsx:8]: > enter

?

**Review by w00tc — APPROVED** [review]

**Pull request merged**